prep_data.py

```diff
@@ -0,0 +1,66 @@
+__author__ = "Keenan Manpearl"
+__date__ = "2023/03/01"
+
+"""
+Create a mitocheck dataset for input into pytorch model
+"""
+import cv2
+import numpy as np
+from torch.utils.data import Dataset
+
+
+class Mitocheck_Dataset(Dataset):
```

Comment:
Consider using [numpy docstrings](https://numpydoc.readthedocs.io/en/latest/format.html) to make documentation more clear (example below):

![image](https://user-images.githubusercontent.com/105665675/223264021-44e96ffb-eb84-4100-9674-71547c0a5c8e.png)